internal/artifacts/meta_test.go

```diff
@@ -276,6 +276,14 @@ func (loc *testLoc) GetSourceModuleArtifactRelPath(modulePath, artifactPath stri
 	return loc.loc.GetSourceModuleArtifactRelPath(modulePath, artifactPath)
 }
 
+func (loc *testLoc) GetTargetTmpDir() string {
```

Comment:
If you remove `GetTargetTmpDir` from the interface, remove this method too

Reply:
removed